save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/fossgraph/VulnerabilityInfoTab.kt

```diff
@@ -0,0 +1,238 @@
+@file:Suppress("FILE_NAME_INCORRECT", "FILE_NAME_MATCH_CLASS")
+
+package com.saveourtool.save.frontend.components.views.fossgraph
+
+import com.saveourtool.save.entities.vulnerability.VulnerabilityDto
+import com.saveourtool.save.entities.vulnerability.VulnerabilityProjectDto
+import com.saveourtool.save.entities.vulnerability.VulnerabilityProjectType
+import com.saveourtool.save.frontend.components.modal.displayModal
+import com.saveourtool.save.frontend.components.modal.mediumTransparentModalStyle
+import com.saveourtool.save.frontend.components.tables.TableProps
+import com.saveourtool.save.frontend.components.tables.columns
+import com.saveourtool.save.frontend.components.tables.tableComponent
+import com.saveourtool.save.frontend.components.tables.value
+import com.saveourtool.save.frontend.externals.fontawesome.faTrashAlt
+import com.saveourtool.save.frontend.utils.*
+
+import react.*
+import react.dom.html.ReactHTML.div
+import react.router.dom.Link
+import web.cssom.ClassName
+
+import kotlinx.serialization.encodeToString
+import kotlinx.serialization.json.Json
+import react.dom.html.ReactHTML.td
+
+val renderVulnerabilityInfo: FC<VulnerabilityInfoTabProps> = FC { props ->
```

Comment:
pls change the name in future PR. Component name should not start with render, as it is function prefix

Reply:
Renamed